src/sentry/snuba/referrer.py

```diff
@@ -562,6 +564,7 @@ class Referrer(Enum):
     API_TRACE_VIEW_SPAN_DETAIL = "api.trace-view.span-detail"
     API_TRACE_VIEW_COUNT_PERFORMANCE_ISSUES = "api.trace-view.count-performance-issues"
     API_TRACE_VIEW_GET_PARENTS = "api.trace-view.get-parents"
+    API_TRACE_VIEW_GET_OCCURRENCE_IDS = "api.trace-view.get-occurrence-ids"
```

Comment:
I'm surprised this was in code by not in the referrer. I thought we throw an exception when a referrer is unknown to snuba